common/vospi.c

```diff
@@ -213,12 +215,18 @@ int vospi_snapshot(uint32_t timeout_ms) {
     // Snapshot start tick
     mp_uint_t tick_start = mp_hal_ticks_ms();
 
-    do {
+    for (;;) {
         if (vospi.flags & VOSPI_FLAGS_RESYNC) {
             vospi.flags &= ~VOSPI_FLAGS_RESYNC;
             vospi_resync();
         }
 
+        MICROPY_EVENT_POLL_HOOK
```

Comment:
Should not use this anymore, there are new functions for this in runtime.h I think.